packages/schema/lib/functional-constraints/bulkWriteConstraints.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const _ = require('lodash');
+const jsonschema = require('jsonschema');
+
+const bulkWriteConstraints = (definition) => {
+  const errors = [];
+  const actionType = 'creates';
+
+  if (definition[actionType]) {
+    _.each(definition[actionType], (actionDef) => {
+      if (actionDef.operation && actionDef.operation.bulk) {
+        if (!actionDef.operation.performBulk) {
```

Comment:
The other direction should be checked too: If `performBulk` is defined, `bulk` should be defined too.

Reply:
Done